lib/accelerator.js

```diff
@@ -0,0 +1,140 @@
+/*!
+ * Veres One Accelerator API.
+ *
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+const _ = require('lodash');
+const async = require('async');
+const bedrock = require('bedrock');
+const config = bedrock.config;
+const didv1 = require('did-veres-one');
+const fs = require('fs');
+const forge = require('node-forge');
+const logger = require('./logger');
+
+require('bedrock-express');
+
+// module API
+const api = {};
+module.exports = api;
+
+bedrock.events.on('bedrock-express.ready', (app, callback) =>
+  _setupAccelerator(callback));
+
+// try to read and process, or generate save and process
+function _readOrCreateJson({path, create, loaded, stored}, callback) {
+  // try to read
+  fs.readFile(path, 'utf8', (err, data) => {
+    if(err) {
+      if(err.code === 'ENOENT') {
```

Comment:
Code doesn't have to indented so much if you bail early on `!(err && err.code === 'ENOENT')`.

Reply:
Can do a TODO for that, doesn't need to be done right away.